charts/linkerd-control-plane/templates/destination.yaml

```diff
@@ -125,6 +125,12 @@ spec:
 {{ $_ := set $tree.Values.proxy "workloadKind" "deployment" -}}
 {{ $_ := set $tree.Values.proxy "component" "linkerd-destination" -}}
 {{ $_ := set $tree.Values.proxy "waitBeforeExitSeconds" 0 -}}
+{{- if not (empty .Values.destinationProxyResources) }}
+{{- $c := dig "cores" .Values.proxy.cores .Values.destinationProxyResources }}
+{{- $_ := set $tree.Values.proxy "cores" $c }}
+{{- $r := merge .Values.destinationProxyResources .Values.proxy.resources }}
+{{- $_ := set $tree.Values.proxy "resources" $r }}
+{{- end }}
```

Comment:
IIUC, although this move wasn't strictly necessary it's cleaner to have these on a more global "scope" given it applies to both the proxy and linkerd-init, right? If so, why not doing the same for the identity workload?

Reply:
The simple answer is...I forgot 😅 good spot! fixed now.